docs/understanding-tracking-design/managing-tracking-scenarios-api/index.md

```diff
@@ -0,0 +1,367 @@
+        "source":"event.schema"
+    }
+]
+...
+```
+
+:::
+
+We will discuss each case, explaining the request type, use case scenarios, and the different content that these fields may have.
+
+## Retrieve a List of Tracking Scenarios
+
+Use this request to retrieve a list of scenarios within an organization, which will be wrapped into the `data` field of the [response](#response-format).
+
+`GET /api/msc/v1/organizations/{organizationId}/tracking-scenarios/v1`
+
+### Query Parameters and Filters
+
+You can filter the results based on the following query parameters:
+
+- `dataStructureId`: Filters the tracking scenarios associated with a particular data structure.
+- `dataStructureVersion`: Filters the tracking scenarios associated with a specific data structure version when used with `dataStructureId`.
+- `withHistory`: Returns a list with the history of each tracking scenario in the `includes` field of the response. The relation between tracking scenarios in `data` and history in `includes` is `id = scenarioId`.
+- `status`: Filters the tracking scenarios that match the specified status.
+
+### Compatibility Checks
+
+This request may return warnings within the `errors` field of the response:
+
+- **SchemaIncompatible**: The tracking scenario is not compatible with a specific Iglu schema version. The `type` is always `Warning`.
+- **SchemaUndecidable**: It cannot determine the compatibility of the tracking scenario with a specific Iglu schema version. The `type` is always `Warning`.
+
+## Retrieve a Specific Tracking Scenario
+
+Use this request to retrieve a specific tracking scenario within an organization, which will be wrapped into the `data` field of the response.
+
+`GET /api/msc/v1/organizations/{organizationId}/tracking-scenarios/v1/{trackingScenarioId}`
+
+### Query Parameters and Filters
+
+You can filter the results based on the following query parameters:
+
+- `withHistory`: Returns a list with the history for the tracking scenario in the `includes` field of the response, related to the tracking scenario by its ID.
+- `status`: Filters the tracking scenarios that match the specified status.
+
+## Creating a Tracking Scenario
+
+Use this request to create a tracking scenario within an organization.
+
+`POST /api/msc/v1/organizations/{organizationId}/tracking-scenarios/v1`
+
+### Request and Response
+
+The creation form has two fields at the top level, as shown in this example:
+
+```
+{
+    "scenario": {
+        "name": "Search",
+        "description": "Tracking the use of the search box",
+        "event": {
+            "source": "iglu:com.example/ui_actions/jsonschema/1-0-0"
+        }
+    },
+    "message": "initial draft"
+}
+```
+
+- `message`: An optional field to provide a message.
+- `scenario`: The definition of the tracking scenario, which should have a defined structure. See [Validations](#validations).
```

Comment:
I don't understand the point about having "a defined structure", is this a reference to the schema referenced by `event.source`?

Reply:
Hmm not sure what you mean Costas, can you elaborate a little bit on this? Thank you 😅